.gemini/styleguide.md

```diff
@@ -0,0 +1,715 @@
+## Key Principles
```

Comment:
You had a checklist for reviewing, maybe we can add that to this file too?

Reply:
those points are incorporated here